modules/aws_roles/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+
+module rosa_operator_roles {
```

Comment:
Why to use module inside another module, instead of adding all the resources to the first one?